src/main/java/core/basesyntax/dao/FruitDaoImpl.java

```diff
@@ -0,0 +1,22 @@
+package core.basesyntax.dao;
+
+import core.basesyntax.db.Storage;
+import java.util.Map;
+
+public class FruitDaoImpl implements FruitDao {
+
+    @Override
+    public Map<String, Integer> getStorage() {
```

Comment:
Direct access to the storage map violates the encapsulation principle. Consider adding methods to handle the logic of accessing and modifying the storage, rather than exposing the storage directly.